packages/coding-standards/src/Sniffs/ForceLateStaticBindingForProtectedConstantsSniff.php

```diff
@@ -4,11 +4,16 @@
 
 namespace Shopsys\CodingStandards\Sniffs;
 
+use function in_array;
```

Comment:
are imports of native functions required? 

like do the fixer add import when I write only 
```php
if (!in_array($constantName, $protectedConstants, true)) {
```

Reply:
No really it was importer beacuse of previous usage like

```
if (!\in_array($constantName, $protectedConstants, true)) {
```